crates/pathfinder/src/bin/pathfinder.rs

```diff
@@ -151,26 +151,18 @@ Hint: Make sure the provided ethereum.url and ethereum.password are good.",
 
 /// Verifies that the database matches the expected chain; throws an error if it does not.
 fn verify_database_chain(storage: &Storage, expected: core::Chain) -> anyhow::Result<()> {
-    use pathfinder_lib::consts::{GOERLI_GENESIS_HASH, MAINNET_GENESIS_HASH};
-    use pathfinder_lib::core::StarknetBlockNumber;
+    use pathfinder_lib::storage::StarknetBlocksTable;
 
     let mut connection = storage.connection().context("Create database connection")?;
     let transaction = connection
         .transaction()
         .context("Create database transaction")?;
-    let genesis = pathfinder_lib::storage::StarknetBlocksTable::get(
-        &transaction,
-        StarknetBlockNumber::GENESIS.into(),
-    )
-    .context("Read genesis block from database")?;
 
-    let db_chain = match genesis {
+    let db_chain = match StarknetBlocksTable::determine_chain(&transaction)
```

Comment:
Commit message has StarknetTransactionTable and the &transaction got me confused.

Reply:
I've fixed the commit message.